postgresql/helpers.go

```diff
@@ -288,12 +288,20 @@ func pgArrayToSet(arr pq.ByteaArray) *schema.Set {
 	return schema.NewSet(schema.HashString, s)
 }
 
-func setToPgIdentList(schema string, idents *schema.Set) string {
+func setToPgIdentList(schema string, idents *schema.Set, quoteIdents ...bool) string {
+	useQuotes := true
+	if len(quoteIdents) > 0 {
+		useQuotes = quoteIdents[0]
+	}
```

Comment:
This is one of the ways I've seen to define a default value in a go function — abusing variadic arguments. Feels hacky for sure, but I didn't really want to introduce a whole additional function that was nearly identical.

Reply:
Ehh, I think this is fine for our use-case. Though maybe a comment noting the intention of the variadic might be useful for future reference?